acados/ocp_nlp/allocate_ocp_nlp.h

```diff
@@ -27,8 +27,8 @@ extern "C" {
 #include "acados/ocp_nlp/ocp_nlp_common.h"
 #include "acados/utils/types.h"
 
-void allocate_ocp_nlp_in(int_t N, int_t *nx, int_t *nu, int_t *nb, int_t *nc,
-                         int_t *ng, ocp_nlp_in *const nlp);
+void allocate_ocp_nlp_in(int_t N, int_t *nx, int_t *nu, int_t *nb, int_t *ng,
```

Comment:
I removed it because it wasn't properly implemented in the previous interface. Since nonlinear constraints can cover affine constraints, usability is not affected. I believe that omitting affine constraints, at least for now, relieves some of the burden implementing different sensitivity methods.